backend/apis/leads/add-leads.api.js

```diff
@@ -109,7 +114,7 @@ app.post("/api/leads", (req, res) => {
       return databaseError(req, res, err);
     }
 
-    const leadIds = result.map(data => data.insertId).joint(", ");
```

Comment:
`joint` is not a function -- this API returned a 500 no matter what because of that.